src/sql/parts/dashboard/tabs/dashboardGridTab.contribution.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { IJSONSchema } from 'vs/base/common/jsonSchema';
+import * as nls from 'vs/nls';
+
+import { generateDashboardWidgetSchema } from 'sql/parts/dashboard/pages/dashboardPageContribution';
+import { registerTabContent } from 'sql/platform/dashboard/common/dashboardRegistry';
+
+export const GRID_TABS = 'grid-tab';
```

Comment:
Ideally this would just be a `layout-tab` with a property identifying if it's a grid or flow layout. I think the overlap between the two will be really high

Reply:
I can do this in another PR when we have enough requirement 

---
In reply to: [169729859](https://github.com/Microsoft/sqlopsstudio/pull/740#discussion_r169729859) [](ancestors = 169729859)